db/schema.rb

```diff
@@ -102,8 +102,6 @@
     t.boolean "activated", default: false
     t.datetime "activated_at"
     t.string "remember_digest"
-    t.string "reset_digest"
-    t.datetime "reset_sent_at"
```

Comment:
Sao file migration toàn thêm column mà đến schema lại thành xóa column đi thế này?

Reply:
dạ do để reset_password làm pull sau mà mấy bạn tạo model user có luôn nên em xóa để add migration pull sau ạ